javascript/javascript1/week1/future-age.js

```diff
@@ -0,0 +1,6 @@
+let yearOfBirth=1990;
+let yearFuture=2045;
+
+let age=yearFuture-yearOfBirth;
```

Comment:
try to get used to using `const` by default.
And just use `let` if you decided to change the variable value later.